osu.Game.Rulesets.Taiko/Objects/Drawables/DrawableHitStrong.cs

```diff
@@ -46,18 +44,26 @@ protected override void CheckJudgement(bool userTriggered)
                 Judgement.SecondHit = true;
         }
 
-        protected override bool HandleKeyPress(Key key)
+        public override bool OnReleased(TaikoAction action)
+        {
+            if (action == firstHitAction)
+                firstKeyHeld = false;

```

Comment:
hmm i guess i don't understand what `firstKeyHeld` means then. because i see that as correct behaviour.